geth/jail/extensions/extensions_test.go

```diff
@@ -0,0 +1,116 @@
+package extensions_test
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/jail/extensions"
+	"github.com/status-im/status-go/geth/jail/extensions/console"
+	"github.com/stretchr/testify/suite"
+)
+
+// TestExtensions validates the behaviour of giving conole extensions.
+func TestExtensions(t *testing.T) {
+	suite.Run(t, new(ExtensionsTestSuite))
+}
+
+type ExtensionsTestSuite struct {
+	suite.Suite
+	vm *otto.Otto
+}
+
+// TestConsoleError will validate the operations of the console.error extension
+// for the otto vm.
+func (s *ExtensionsTestSuite) TestConsoleError() {
+	require := s.Require()
+	written := "Bob Marley"
+
+	var customWriter bytes.Buffer
+
+	console.Stderr = &customWriter
+
+	_, err := s.vm.Run(fmt.Sprintf(`
+		console.error(%q);
+	`, written))
+
+	require.NoError(err)
+
+	require.NotEmpty(&customWriter)
+	require.Equal(written, strings.TrimPrefix(customWriter.String(), "console.error: "))
+}
+
+// TestConsoleDebug will validate the operations of the console.debug extension
+// for the otto vm.
+func (s *ExtensionsTestSuite) TestConsoleDebug() {
+	require := s.Require()
+	written := "Bob Marley"
+
+	var customWriter bytes.Buffer
+
+	console.Stddebug = &customWriter
+
+	_, err := s.vm.Run(fmt.Sprintf(`
+		console.debug(%q);
```

Comment:
Ideally, let's have a test where internal object variable is created, gets populated, and then dumped using log. I am even ok if to dump variable you have to use `console.log(JSON.stringify(objectVar))`.

Also: we need to actually check that notification is sent (we can substitute default notification handler to do so, see https://github.com/status-im/status-go/blob/develop/geth/node/signals.go#L50)